pkg/systemd/quadlet/quadlet.go

```diff
@@ -323,6 +328,29 @@ var (
 		KeyVariant:              true,
 	}
 
+	// Supported keys in "Build" group
+	supportedBuildKeys = map[string]bool{
```

Comment:
There are three additional keys all Quadlet files support. I know the correct implementation should handle it in one place, but, it currently doesn't. The keys are:
- `ContainersConfModule` and `GlobalArgs` which are handled by calling `createBasePodmanCommand`. These flags control arguments that are at podman level, not specific to the command
- `PodmanArgs` which is handled by calling `handlePodmanArgs`. This key allows passing arguments that do not have a dedicated Quadlet key 

I see that you are using these functions correctly, but you are not allowing their usage

Reply:
Thanks for this hint. Indeed, I didn't know that these exist and I have to add them to the supported keys manually.

Reply:
I have fixed this in my latest push.